#
# Move /etc/systemd/system content to /usr/lib/systemd/system.
#
--- /etc/systemd/system/dbus.service	2016-06-20 05:18:28.000000000 +0000
+++ /dev/null	2018-10-19 17:49:21.943570148 +0000
@@ -1,9 +0,0 @@
-[Unit]
-Description=D-Bus System Message Bus
-Documentation=man:dbus-daemon(1)
-Requires=dbus.socket
-
-[Service]
-ExecStart=/usr/bin/dbus-daemon --system --address=systemd: --nofork --nopidfile --systemd-activation
-ExecReload=/usr/bin/dbus-send --print-reply --system --type=method_call --dest=org.freedesktop.DBus / org.freedesktop.DBus.ReloadConfig
-
--- /usr/lib/systemd/system/dbus.service	2016-03-17 09:25:11.000000000 +0000
+++ /usr/lib/systemd/system/dbus.service	2016-06-20 05:18:28.000000000 +0000
@@ -6,4 +6,4 @@
 [Service]
 ExecStart=/usr/bin/dbus-daemon --system --address=systemd: --nofork --nopidfile --systemd-activation
 ExecReload=/usr/bin/dbus-send --print-reply --system --type=method_call --dest=org.freedesktop.DBus / org.freedesktop.DBus.ReloadConfig
-OOMScoreAdjust=-900
+

--- /etc/systemd/system/sssd.service.d/journal.conf	2017-07-25 15:11:23.000000000 +0000
+++ /dev/null	2018-10-21 06:25:22.804632497 +0000
@@ -1,7 +0,0 @@
-[Service]
-# Uncomment *both* of the following lines to enable debug logging
-# to go to journald instead of /var/log/sssd. You will need to
-# run 'systemctl daemon-reload' and then restart the SSSD service
-# for this to take effect
-#ExecStart=
-#ExecStart=/usr/sbin/sssd -i
--- /etc/systemd/system/httpd.service	2016-08-19 13:27:11.000000000 +0000
+++ /dev/null	2018-10-19 17:49:21.943570148 +0000
@@ -1,7 +0,0 @@
-.include /usr/lib/systemd/system/httpd.service
-
-[Service]
-Environment=KRB5CCNAME=/var/run/httpd/ipa/krbcache/krb5ccache
-Environment=KDCPROXY_CONFIG=/etc/ipa/kdcproxy/kdcproxy.conf
-ExecStartPre=/usr/libexec/ipa/ipa-httpd-kdcproxy
-ExecStopPost=-/usr/bin/kdestroy -A
--- /dev/null	2018-10-19 17:49:21.943570148 +0000
+++ /usr/lib/systemd/system/httpd.service.d/ipa.conf	2018-10-19 19:16:30.067904539 +0000
@@ -0,0 +1,5 @@
+[Service]
+Environment=KRB5CCNAME=/var/run/httpd/ipa/krbcache/krb5ccache
+Environment=KDCPROXY_CONFIG=/etc/ipa/kdcproxy/kdcproxy.conf
+ExecStartPre=/usr/libexec/ipa/ipa-httpd-kdcproxy
+ExecStopPost=-/usr/bin/kdestroy -A
#
# Move /etc/tmpfiles.d content to /usr/lib/tmpfiles.d
#
--- /etc/tmpfiles.d/opendnssec.conf    2016-02-01 12:14:52.000000000 +0000
+++ /dev/null  2018-09-17 08:06:53.663641878 +0000
@@ -1 +0,0 @@
-D /var/run/opendnssec 0755 ods ods -
--- /dev/null  2018-09-17 08:06:53.663641878 +0000
+++ /usr/lib/tmpfiles.d/opendnssec.conf        2016-02-01 12:14:52.000000000 +0000
@@ -0,0 +1 @@
+D /var/run/opendnssec 0755 ods ods -
#
# Avoid any default dependencies for targets that will be used.
#
--- /usr/lib/systemd/system/sysinit.target	2016-02-01 14:04:05.000000000 +0000
+++ /usr/lib/systemd/system/sysinit.target	2018-10-19 20:05:31.937494953 +0000
@@ -9,5 +9,5 @@
 Description=System Initialization
 Documentation=man:systemd.special(7)
 Conflicts=emergency.service emergency.target
-Wants=local-fs.target swap.target
+### Wants=local-fs.target swap.target
 After=local-fs.target swap.target emergency.service emergency.target
--- /usr/lib/systemd/system/basic.target	2016-05-30 04:23:03.000000000 +0000
+++ /usr/lib/systemd/system/basic.target	2018-10-21 06:23:01.878251807 +0000
@@ -9,7 +9,7 @@
 Description=Basic System
 Documentation=man:systemd.special(7)
 Requires=sysinit.target
-Wants=sockets.target timers.target paths.target slices.target
+### Wants=sockets.target timers.target paths.target slices.target
 After=sysinit.target sockets.target paths.target slices.target tmp.mount
 
 # We support /var, /tmp, /var/tmp, being on NFS, but we don't pull in
#
# Workaround 1390191 in read-only container.
#
--- /usr/lib/tmpfiles.d/etc.conf	2016-05-30 04:23:04.000000000 +0000
+++ /usr/lib/tmpfiles.d/etc.conf	2018-11-30 05:54:18.935849859 +0000
@@ -9,6 +9,6 @@
 
 L /etc/os-release - - - - ../usr/lib/os-release
 L /etc/localtime - - - - ../usr/share/zoneinfo/UTC
-L+ /etc/mtab - - - - ../proc/self/mounts
+L /etc/mtab - - - - ../proc/self/mounts
 C /etc/nsswitch.conf - - - -
 C /etc/pam.d - - - -
